hms-app/src/pages/Archive.tsx

```diff
@@ -1,18 +1,74 @@
-import React from 'react'
-import ideaData from '../test/TestIdeaData'
-import IdeaCardList from '../components/IdeaCardList'
-import { Idea } from '../common/types'
+import React, { useEffect, useState } from 'react'
+import { getListOfHackathons } from '../actions/GetBackendData'
+import HackathonDetails from '../components/HackathonDetails'
+import { Select } from '@mantine/core'
+
+interface HackathonPreview {
+  id: string
+  title: string
+}
 
 export default function Archive() {
+  const [selectedHackweek, setSelectedHackweek] = useState('')
+  const [hackathonList, setHackathonList] = useState({
+    errorhackathonList: false,
+    isLoadinghackathonList: true,
+    hackathons: [] as HackathonPreview[],
+  })
+
+  const { errorhackathonList, isLoadinghackathonList, hackathons } =
+    hackathonList
+
+  const loadHackathons = () => {
+    getListOfHackathons('hackathons').then(
+      (data) => {
+        setHackathonList({
+          ...hackathonList,
+          hackathons: data.hackathons,
+          errorhackathonList: false,
+          isLoadinghackathonList: false,
+        })
+      },
+      () => {
+        setHackathonList({
+          ...hackathonList,
+          errorhackathonList: true,
+          isLoadinghackathonList: false,
+        })
+      }
+    )
+  }
+
+  const data = hackathonList.hackathons.map(
+    (hackathon, index) => `${hackathon.id}`
+  )
+
+  const selectChange = (value: string) => {
+    setSelectedHackweek(value)
+  }
+
+  useEffect(() => {
+    loadHackathons()
+  }, [])
+
   return (
     <>
-      <h1>this is the Archive</h1>
-      <h2>Ideas List:</h2>
-      <IdeaCardList
-        ideas={ideaData as Idea[]}
-        columnSize={6}
-        type={'archive'}
-      />
+      {isLoadinghackathonList && <div>hackathon select is loading...</div>}
+      {!isLoadinghackathonList && (
+        <div style={{ border: '1px solid red', width: 250 }}>
+          <Select
+            placeholder={'select a Hackweek'}
```

Comment:
Probably says `Hackweek` in other places as well

```suggestion
            placeholder={'select a Hackathon'}
```